### DISCUSSION TITLE
还是建议关闭讨论

### DISCUSSION THREAD

**Discussion by acclea** [WeNeedHome/SummaryOfLoanSuspension#1026] [General]
有太多的苍蝇，它们活着就是为了恶心所有人

**Comment by xujiyou** [comment]
你这跟政府有啥区别呢，都来到墙外面了，还是把嘴给缝上，自我阉割

**Comment by fnnnnng** [comment]
脑瘫东西 建议你好好跪着别出来丢人现眼

**Comment by clleo888** [comment]
其实想一想, 不如去关闭老共，党内太多的苍蝇，它们活着就是为了恶心所有人,相信你会非常支持!

**Comment by TALK1949** [comment]
因为发现自己在墙内颠簸不破的真理出了墙却没有了市场，感到极度不适而已。
没有删帖禁言五毛加持的舆论环境，慢慢适应吧，面对现实。

**Comment by volans19** [comment]
你就是党派来的狗？

**Comment by fanren112233** [comment]
大孝子

**Comment by oceanzhang860411** [comment]
看不慣你可以回你的牆內，抖音，b站，微博可以滿足你，讓你爽翻天

**Comment by toxin08x** [comment]
如果你是烂尾楼业主，那我只能说你活该，祝福你以后一直买到烂尾楼。如果你不是烂尾楼业主，那你就是坏，并且站在这些维权业主的对立面。并且你还挺自大的，自己看着不同观点不爽就能代表所有人？你这种思维建议回墙内吧，这里不适合你，何必要继续恶心自己呢？

**Comment by odoo-compose** [comment]
> 有太多的苍蝇，它们活着就是为了恶心所有人

请留在国内媒体玩。

**Comment by itsoursduty** [comment]
自我阉割且妄图阉割别人的奴才

**Comment by FlyMeToTheMars911** [comment]
狗奴才
